.circleci/config.yml

```diff
@@ -50,7 +50,38 @@ jobs:
             - node_modules
             - build
             - dist
-            - .cache/Cypress
+            - assets/stylesheets
+
+  build-int-test:
+    executor:
+      name: hmpps/node
+      tag: << pipeline.parameters.node-version >>
+    steps:
+      - checkout
+      - run:
+          name: Update npm
+          command: 'sudo npm install -g npm@9.8.1'
```

Comment:
Can this be unpinned now? So use `sudo npm install -g npm@latest`?

Reply:
Think there was an issue where we had to pin it to 9.8.1, but hopefully that's resolved now? https://mojdt.slack.com/archives/C01AHT2AMCJ/p1693553887117429

Reply:
Updated